scipy/sparse/_sputils.py

```diff
@@ -388,6 +390,27 @@ def is_pydata_spmatrix(m) -> bool:
     return base_cls is not None and isinstance(m, base_cls)
 
 
+def convert_pydata_sparse_to_scipy(
```

Comment:
I'd prefer to have a function that takes a single input and returns a single output. In cases where we need to convert multiple objects, we can call the function multiple times.

Reply:
Sure! Now the function accepts single input only.